### PR TITLE
feat: convex deploy

### PR DESCRIPTION
**Pull request by mds1**
Still need to do the actual deploy before merging

### CODE REVIEW

File: scripts/create-protection-market-convex.ts [hunk 1]
```diff
@@ -0,0 +1,140 @@
...
+  // We know that Money Markets have a trigger address of the zero address, so we use that to query the Comptroller
+  // for the Money Market address
+  const comptrollerAddress = getContractAddress('Comptroller', chainId);
+  const comptroller = new Contract(comptrollerAddress, comptrollerAbi, signer); // connect signer for sending transactions
+  const cozyMMAddress = await comptroller.getCToken(underlyingAddress, AddressZero);
+
+  // If the returned address is the zero address, a money market does not exist and we cannot deploy a protection
+  // market with the desired underlying
+  if (cozyMMAddress === AddressZero) {
+    logFailure('Money Market exists. Exiting script');
```
Comment (lvwrence) [inline scripts/create-protection-market-convex.ts]: this should be money market *doesn't* exist

**Review by lvwrence — APPROVED** [review]

File: scripts/create-protection-market-convex.ts (hunk 1 above)
Comment (mds1) [reply scripts/create-protection-market-convex.ts]: Oops, fixed in 69755fb

File: scripts/create-protection-market-convex.ts [hunk 2]
```diff
@@ -0,0 +1,140 @@
...
+
+// STEP 0: ENVIRONMENT SETUP
+const provider = hre.ethers.provider;
+const signer = new hre.ethers.Wallet(process.env.PRIVATE_KEY as string, hre.ethers.provider);
+const chainId = getChainId(hre);
+
+// STEP 1: TRIGGER CONTRACT SETUP
+const platformIds = [10];
+// const recipient = signer.address;
+const recipient = '0xSetRecipientAddressHere'; // subsidy recipient
```
Comment (apbendi) [inline scripts/create-protection-market-convex.ts:29]: Doesn't this need to be set?

**Review by apbendi — APPROVED** [review]
Looks good. One minor comment.

File: scripts/create-protection-market-convex.ts (hunk 2 above)
Comment (mds1) [reply scripts/create-protection-market-convex.ts:29]: It's intentionally left unset in all deploy scripts to (1) help prevent accidental deploys and (2) ensure the deployer remembers to change the recipient address to the correct one before deploying

File: scripts/create-protection-market-convex.ts (hunk 1 above)
Comment (mds1) [reply scripts/create-protection-market-convex.ts]: fixed in 69755fb

**Pull request merged**